systemtest/src/main/java/io/strimzi/systemtest/utils/kafkaUtils/KafkaUtils.java

```diff
@@ -645,4 +648,68 @@ public static void verifyKafkaKraftMetadataLog(final TestStorage testStorage, fi
     private static String buildDirectoryPath(int volumeId, int kafkaIndex) {
         return String.format("/var/lib/kafka/data-%d/kafka-log%d/__cluster_metadata-0", volumeId, kafkaIndex);
     }
+
+    /**
+     * Waits until the Kafka cluster in the given namespace the specified AutoRebalance state is set.
+     *
+     * @param namespaceName         The name of the Kubernetes namespace where the Kafka cluster resides.
+     * @param clusterName           The name of the Kafka cluster.
+     * @param expectedState         The expected {@link KafkaAutoRebalanceState} that is set.
+     */
+    public static void waitUntilKafkaHasAutoRebalanceState(final String namespaceName,
+                                                           final String clusterName,
+                                                           final KafkaAutoRebalanceState expectedState) {
+        TestUtils.waitFor(
+            String.format("waiting for Kafka cluster '%s' in namespace '%s' and KafkaRebalance state: '%s'", clusterName, namespaceName, expectedState),
```

Comment:
```suggestion
            String.format("Kafka cluster: '%s' in Namespace: '%s' and KafkaRebalance state: '%s'", clusterName, namespaceName, expectedState),
```
the "wait for" or "waiting for" sentence is already as part of the log